sherpa/ui/tests/test_session.py

```diff
@@ -248,3 +251,22 @@ def test_list_model_ids():
 
     s.set_model('const1d.mdla')
     assert set(s.list_model_ids()) == set([1, 'ma', 'mb'])
+
+
+# Fix 476
+def test_zero_division_calc_stat():
+    ui = AstroSession()
+    x = numpy.arange(100)
```

Comment:
Interesting that haven't needed to `import numpy` before adding this test.  If you wanted to avoid it you can also just do

```python
x = list(range(100))
y = [0]*100
```

[That is lists' work too.]

Reply:
By the time the test is run numpy has been probably imported so many times already, I don't think it matters really.